File: Demo/setter属性测试/setter属性测试/ViewController.m
//
//  ViewController.m
//  setter属性测试
//
//  Created by kadis on 2019/8/23.
//  Copyright © 2019 https://github.com/GGTechnology. All rights reserved.
//

#import "ViewController.h"
#import "views.h"

@interface ViewController ()

@end

@implementation ViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view.
    views *view = [views new];
    view.testArray = @[@"string", @1];
    view.testDic = @{@"name":@"mike", @"age":@17};
}


@end

File: Demo/SWIFT/对象、图片、NSData的 本地缓存/ZZDiskCache/ViewController.swift
//
//  ViewController.swift
//  ZZDiskCache
//
//  Created by duzhe on 16/3/2.
//  Copyright © 2016年 dz. All rights reserved.
//

import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.

       let homePath = NSHomeDirectory()
        
       print(homePath)
        
       let stu = Student()
       stu.name = "小王"
       stu.id = 1
        `$`.saveObj("xxxx", value: stu)
        
        `$`.getObj("xxxx") { (obj) -> () in
         if let obj = obj as? Student{
            print("\(String(describing: obj.id)) , \(obj.name)")
         }
       }
//        ssdfweefweslwhlfhawkh
    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }

}

